assets/panel/Histograms.json

```diff
@@ -0,0 +1,10 @@
+{
+  "DEVTOOLS_DEBUGGER_LOAD_SOURCE": {
+    "record_in_processes": ["main", "content"],
+    "expires_in_version": "never",
+    "kind": "exponential",
+    "high": 10000,
+    "n_buckets": 1000,
+    "description": "The time (in milliseconds) that it took to load a source for the user."
```

Comment:
This looks like Type-1 or Type-2 data. Do we want it to be opt-in or opt-out on the Release channel?

Reply:
I'm not too sure... how are we handling this on other tools? @juliandescottes ?

Reply:
Most likely type 2 data category ([data collection categories](https://wiki.mozilla.org/Firefox/Data_Collection#Data_Collection_Categories)). 

Regarding opt-in/opt-out, I have never explicitly defined this in Histograms.json, so not sure either. There has been a recent data collection policy change ([link](https://medium.com/georg-fritzsche/data-preference-changes-in-firefox-58-2d5df9c428b5)). Don't know if we can still use the releaseChannelCollection property for new probes?

Let's ask @fmarier

Reply:
That looks like Category1/2, which is allowed to be opt-out on all channels (including release).